.travis.yml

```diff
@@ -119,9 +121,13 @@ matrix:
           packages:
             - *native_deps
 
-before_install: pip3 install --user toml $extra_pip
+before_install:
+  - pip3 install --user toml $extra_pip
 
-script: make -j 4 && make check
+script:
```

Comment:
This is better done as a chained `&&` instead of this list.  That is because if one fails, it will still continue and run the next item.